R/choices_selected.R

```diff
@@ -244,8 +261,7 @@ vector_reorder <- function(vec, idx) {
   }
 
   attributes(vec) <- vec_attrs
-
-  return(vec)
+  vec
 }
 
 vector_pop <- function(vec, idx) {
```

Comment:
I think we should explicitly add theses tags before utils function definition 
```suggestion
#' @noRd
#' @keyword internal
vector_pop <- function(vec, idx) {
```

Reply:
Done in 5887a0b1 I think I caught all